mappings/net/minecraft/fluid/BaseFluid.mapping

```diff
@@ -6,6 +6,9 @@ CLASS net/minecraft/class_3609 net/minecraft/fluid/BaseFluid
 		ARG 1 world
 		ARG 2 fluidPos
 		ARG 3 state
+	METHOD method_15726 (Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;)Ljava/util/Map;
```

Comment:
I suggest naming this method if possible

Reply:
I'm working on this, might take a little bit of time tho.